CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.15)
+project(HGS_CVRP)
+set(CMAKE_CXX_STANDARD 11)
+
+set(src_files
+        Program/Genetic.cpp
+        Program/Individual.cpp
+        Program/LocalSearch.cpp
+        Program/Params.cpp
+        Program/Population.cpp
+        Program/Split.cpp
+        )
+
+include_directories(Program)
+#include_directories(Interface)
+
+# Build Executable
+add_executable(bin
+        Program/main.cpp
+        ${src_files}
+        )
+set_target_properties(bin PROPERTIES OUTPUT_NAME genvrp)
+
+# Test Executable
+enable_testing()
+add_test(NAME    test_bin
+         COMMAND genvrp ../Instances/CVRP/X-n101-k25.vrp mySolution.sol -seed 1)
```

Comment:
Perhaps naming the executable ./hgs could be more transparent.
./genvrp is a legacy name that does not represent much anymore (not a real terminology in the paper/project)

Reply:
Renaming it to ./hgs makes sense.